cpp/include/cudf/utilities/type_dispatcher.hpp

```diff
@@ -168,7 +167,7 @@ CUDF_TYPE_MAPPING(uint32_t, type_id::UINT32)
 CUDF_TYPE_MAPPING(uint64_t, type_id::UINT64)
 CUDF_TYPE_MAPPING(float, type_id::FLOAT32)
 CUDF_TYPE_MAPPING(double, type_id::FLOAT64)
-CUDF_TYPE_MAPPING(cudf::string_view, type_id::STRING)
+CUDF_TYPE_MAPPING(bool, type_id::BOOL8)
```

Comment:
I moved these around so that the order matches the order of the enum definitions in `type_id`.